quinn-proto/src/crypto/rustls.rs

```diff
@@ -256,6 +256,7 @@ impl crypto::ClientConfig for rustls::ClientConfig {
         params: &TransportParameters,
     ) -> Result<Box<dyn crypto::Session>, ConnectError> {
         let version = interpret_version(version)?;
+        // TODO: why is this unwrap safe?
```

Comment:
Is this safe, and if so, why?

Reply:
Seems to me that this could panic if someone sets a funny provider. Maybe we should return an error here, similar to how `interpret_version` does above?